src/game/level.c

```diff
@@ -930,8 +940,12 @@ bool Level_Load(int level_num)
         g_GameFlow.levels[level_num].injections.length,
         g_GameFlow.levels[level_num].injections.data_paths, m_InjectionInfo);
 
-    bool ret =
-        Level_LoadFromFile(g_GameFlow.levels[level_num].level_file, level_num);
+    bool is_demo =
+        (g_GameFlow.levels[level_num].level_type == GFL_TITLE_DEMO_PC)
+        | (g_GameFlow.levels[level_num].level_type == GFL_LEVEL_DEMO_PC);
```

Comment:
Rather than a binary op we can use `||`. Also, we can mark the variables as `const`.